<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPanelDialog</class>
 <widget class="QDialog" name="AddPanelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Panel</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>210</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel position</string>
   </property>
  </widget>
  <widget class="QComboBox" name="position">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Top</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bottom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="screenLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>210</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="screen">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="componentsLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>68</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Optional features</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showApplicationMenu">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Application Menu</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showPager">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Pager</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Please set up your new panel.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showClock">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Clock</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTaskManager">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Show running tasks</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPanelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPanelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
